Feat/centered large asset

AlyshaNolan opened the pull request:
# Context

[Link to Notion ticket](https://www.notion.so/cloudcannon/Centered-Large-Asset-4297906506594b9d8ee453fb80111636?pvs=4)

# Testing (tester)

The reviewer should check on this branch:

- The code
- The component in Bookshop browser
- The site in CloudCannon ([link here](https://app.cloudcannon.com/42158/editor#sites/118586/collections/pages/:/edit?editor=visual&url=%2F&path=%2Fsrc%2Fpages%2Findex.html&collection=pages))

# Before merge (PR owner)

- [x] Delete the test site in CloudCannon
- [x] For "generic" components **only**: remove `- content_blocks` from `structures` in the YAML file (this was added for testing).

AlyshaNolan committed:
Feat: create centered large asset component

AlyshaNolan committed:
Merge branch 'main' into feat/centered-large-asset

AlyshaNolan committed:
Feat: add button and asset type

AlyshaNolan committed:
Fix: icon name

AlyshaNolan committed:
Feat: move alignment and color config to be global

AlyshaNolan committed:
Updated 1 file via CloudCannon.

adon-cloudcannon commented:
@AlyshaNolan this looks great - as per our discussion, I will make these changes:

- removed the content_alignment preview option from the yml file
- moved the if statements in the HTML outside of the elements, so that they disappear as well, not just the content

Otherwise, happy to accept :)

Commit:
Refactor: removed unused config option from yml file, altered if statements in HTML to wrap elements and not jsut content

- the HTML refactor brings the heading and asset elements in line with how the button functions - if the heading content or asset is missing, the entire element is removed from the HTML
- previously, only the content was removed and this left an empty element in the block - this was fine except for the flex gap adding some extra unwanted space

adon-cloudcannon reviewed (approved):
Changes approved

AlyshaNolan committed:
Chore: remove content from page
